docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -227,4 +227,7 @@ Then the monitor will resume running according to the defined schedule.
 Using `retestOnFailure` can reduce noise related to transient problems.
 // end::monitor-config-options[]
 
-For information on configuring monitors individually, refer to <<synthetics-monitor-use>>.
+For information on configuring monitors individually, refer to:
+
+* <<synthetics-monitor-use>> for browser monitors 
+* <<synthetics-lightweight-data-types>> for lightweight monitors
```

Comment:
Should this be synthetics-lightweight.html?

Reply:
Good catch! 

```suggestion
* <<synthetics-lightweight>> for lightweight monitors
```